### ISSUE TITLE
Simplify E2E flow after logging in

### ISSUE THREAD

**Issue by soloturn** [element-hq/element-web#21652]
### Steps to reproduce

depending on how you first register, one gets 3 screens. it could, and imo should, be a single one.

### Outcome

![Security key dialog](https://user-images.githubusercontent.com/825568/161398487-db6de2c8-a625-414d-a8c6-7e1e74498d22.png)

---

![Verify this device dialog](https://user-images.githubusercontent.com/825568/161398496-3ab8c729-6603-4bb3-85ce-41c2d59b9eb0.png)

---

![Security phrase dialog](https://user-images.githubusercontent.com/825568/161398517-bd0f979c-b319-4fb5-8382-e5a7b9372ff4.png)

### Will you send logs?

No

**Comment by goelesha** [comment]
@SimonBrandner Can you pls guide me about this issue and assign it to me. I would like to work on it.

**Comment by SimonBrandner** [comment]
> @SimonBrandner Can you pls guide me about this issue and assign it to me. I would like to work on it.

This issue requires input from the design team, so I would recommend choosing a different issue as this might take quite some time for the design team to work through

**Comment by soloturn** [comment]
@SimonBrandner @goelesha, @gsouquet the screen should list the 3 options:

1. register device with pass phrase, input field.
2. register device by uploading security key, input field or file upload.
3. reset all

this screen gets displayed under a title "verify this device". is @goelesha allowed to make a propsal in case she has a good idea?

**Comment by SimonBrandner** [comment]
> is @goelesha allowed to make a propsal in case she has a good idea how to hook something in architecturally proper?

They surely can share their thoughts but the decision is on the design team

**Comment by goelesha** [comment]
@SimonBrandner @soloturn I would still like to work on this. I will try my best to fix it up.

**Comment by richvdh** [comment]
I don't think this issue is accurate. If it is, please provide repro steps for how to get to each screen.
